intl.emu

```diff
@@ -299,11 +294,9 @@
           1. Else if _x_ is ~negative-infinity~, then
             1. Let _n_ be an ILD String value indicating negative infinity.
           1. <ins>Else if _x_ is a Decimal128 value, then</ins>
-            1. <ins>Let _v_ be cohort(_x_).</ins>
-            1. <ins>Let _q_ be quantum(_x_).</ins>
-            1. <ins>If _numberFormat_.[[Style]] is *"percent"*, set _x_ to « _v_ × 100, _q_ + 2 »<sub>𝔻</sub>.</ins>
+            1. <ins>If _numberFormat_.[[Style]] is *"percent"*, set _x_ to _x_ × 100.</ins>
             1. <ins>Set _exponent_ to the truncated exponent _x_.</ins>
-            1. <ins>Set _x_ to ! _v_ × 10<sup>-_exponent_</sup>.</ins>
+            1. <ins>Set _x_ to ! _x_ × 10<sup>-_exponent_</sup>.</ins>
```

Comment:
Also here, can we have "special" values?

Reply:
We can't. Good catch. I've handled that now.